rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java

```diff
@@ -185,6 +186,16 @@ public boolean isWalletEnabled() {
         return getBoolean("wallet.enabled", false);
     }
 
+    public double gasPriceMultiplier() {
+        double gasPriceMultiplier = getDouble(RPC_GAS_PRICE_MULTIPLIER_CONFIG, 1.1);
+
+        if(gasPriceMultiplier > 1.0) {
```

Comment:
just thinking out loud.. probably it makes sense to restrict negative values. But should we really reject values lower than 1 (which basically would mean not increasing but decreasing by some percentage)?

Reply:
I was under the impresion that there would not be a case when the buffer would decrease de value. But if you think we should allow it I can make the change quickly.

Reply:
Done